TG-Netflix/src/components/lane/SlideContent.jsx

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+
+const Container = styled.div`
```

Comment:
Ik zou de styling weer in een aparte file zetten, want nu vind ik het verwarrend dat er een aparte style file is voor de lanes, maar niet voor de slidecontent. Dat maakt het wat onoverzichtelijker.